### PR TITLE
Add jwt remote option http method

### PR DESCRIPTION
**Pull request by fAuernigg**
* Add option to select http method.

### CODE REVIEW

File: backends/jwt_remote.go [hunk 1]
```diff
@@ -63,6 +65,12 @@ func NewRemoteJWTChecker(authOpts map[string]string, options tokenOptions, versi
 		}
 	}
 
+	if httpMethod, ok := authOpts["jwt_http_method"]; ok {
+		if httpMethod == "POST" || httpMethod == "GET" || httpMethod == "PUT" {
```
Comment (iegomez) [inline backends/jwt_remote.go:69]: This is my bad as I never did before, but could I bother you to refactor the backend to use the `http` constants for methods?

**Review by iegomez — COMMENTED** [review]
Sorry for the belated review.
The PR makes sense to me, but I'd like if we could introduce the same changes to the JWT remote one given their similarities.
Do you think you could add that?

File: backends/jwt_remote.go (hunk 1 above)
Comment (iegomez) [reply backends/jwt_remote.go:69]: @fAuernigg I gather you lost interest in this PR: sorry, that's my bad for taking too long. Let me know if there's something you want to add or edit, elsewise I'll apply the same changes to the `http` backend (at the next point I get some time to do so) and merge.

Cheers!

Comment (fAuernigg) [reply backends/jwt_remote.go:69]: Sorry I missed to read your comment.
I will try to add the method to the http backend.

**Comment by iegomez** [comment]
@fAuernigg I based on your PR and created https://github.com/iegomez/mosquitto-go-auth/pull/233 which just changes string literals to be taken from the http package constants, and adds a couple of tests. Hope you don't mind!

**Pull request closed without merging**